pre-docs/bolt_installing_modules.md

```diff
@@ -42,28 +42,29 @@ For modules that require Ruby gems, see [Installing Gems with Bolt Packages](bol
 
 For more details about specifying modules in a Puppetfile, see the [Puppetfile documentation](https://puppet.com/docs/pe/2018.1/puppetfile.html).
 
-1.   Create a file named Puppetfile and store it in the Boltdir, which can be a directory named Boltdir at the root of your project or a global directory in `$HOME/.puppetlabs/bolt`. 
-2.   Open the Puppetfile in a text editor and add the modules and versions that you want to install. If the modules have dependencies, list those as well. 
+1.   Create a file named Puppetfile and store it in the (bolt project directory)[./bolt_project_directory.md]
+2.   Open the Puppetfile in a text editor and add the modules and versions that you want to install. If the modules have dependencies, list those as well.
 
      ```
      # Modules from the Puppet Forge.
      mod 'puppetlabs/package', '0.2.0'
      mod 'puppetlabs/service', '0.3.1'
-    
+
      # Module from a Git repository.
      mod 'puppetlabs/puppetlabs-facter_task', git: 'git@github.com:puppetlabs/puppetlabs-facter_task.git', ref: 'master'
      ```
 
-3.   Add any task or plan modules stored locally in Boltdir to the list. If these modules are not listed in the Puppetfile, they will be deleted. 
+3.   Add any task or plan modules stored locally in `modules/` to the list. If
+     these modules are not listed in the Puppetfile, they will be deleted.
 
      ```
      mod 'myteam/app_foo', local: true
      ```
 
-     Alternately, any modules you don't want to manage with the Puppetfile can be manually installed to a different subdirectory in the Boltdir, such as `site`.
+     Preferable, any modules you don't want to manage with the Puppetfile will be manually installed to a different subdirecortory. Conventionally, `site-modules` is used for this and is included on the default `modulepath`.
```

Comment:
s/Preferable/Preferably/
s/subdirecortory/sub-directory/

Reply:
Can we be more opinionated about this?

Install any modules that you don't want to manage with the Puppetfile to a different subdirectory. Typically, `site-modules` is used for this purpose and is included on the default `modulepath`.

Reply:
"conventionally" seems more prescriptive than "typically". I included the rest of your suggestion tell me if there is a reason to prefer typically.

Reply:
Conventionally is definitely more prescriptive. I suggested typically only because it's a more common/simple word choice, and conventionally made me wonder *what* conventions? I'm fine with either.

Reply:
Which reminds me I need to talk to jean about: https://puppet.com/docs/pe/2019.0/control_repo.html

Reply:
DOCUMENT-981 and https://github.com/puppetlabs/control-repo/pull/71/files
 are where this convention is defined.